django/db/models/query.py

```diff
@@ -1916,20 +1912,6 @@ def _fetch_all(self):
         if self._prefetch_related_lookups and not self._prefetch_done:
             self._prefetch_related_objects()
 
-    def _next_is_sticky(self):
-        """
-        Indicate that the next filter call and the one following that should
-        be treated as a single filter. This is only important when it comes to
-        determining when to reuse tables for many-to-many filters. Required so
-        that we can filter naturally on the results of related managers.
-
-        This doesn't return a clone of the current QuerySet (it returns
-        "self"). The method is only used internally and should be immediately
-        followed by a filter() that does create a clone.
-        """
-        self._sticky_filter = True
-        return self
```

Comment:
I think this change is now possible now that we've removed the previously existing misuse of this API in 2598b371a93e21d84b7a2a99b2329535c8c0c138 as this one was not immediately filtering after.